htdocs/ecotaxdeee/core/modules/modEcoTaxDeee.class.php

```diff
@@ -188,7 +188,7 @@ function load_tables()
 		global $db;
 
         $sql = array();
-        $sql[] = "CREATE TABLE IF NOT EXISTS llx_ecotax (
+        $sql[] = "CREATE TABLE IF NOT EXISTS llx_ecotaxdeee (
```

Comment:
To create table , you must uncomment the line _load_tables... 2 lines before and create a file llx_ecotaxdeee.sql into the sql dir.